_search-plugins/sql/sql/nested.md

```diff
@@ -0,0 +1,211 @@
+---
+layout: default
+title: Nested Function
+parent: SQL
+grand_parent: SQL and PPL
+nav_order: 13
+redirect_from:
+- /search-plugins/sql/nested/
+---
+
+# Nested
+
+## Nested In SELECT Clause
+
+The nested function is used in the `SELECT` clause to unnest nested object type collections. The nested collection is flattened and a cartesian product is returned when querying against two or more nested collections. The nested function is supported in the V2 engine in the `SELECT` and `WHERE` clauses, see [query-processing-engines](https://opensearch.org/docs/latest/search-plugins/sql/limitation/#query-processing-engines) for more details.
```

Comment:
I feel that phrase
> The nested function is supported

is not suitable here. Maybe you need to move it outside of this paragraph, for example to a separate section on the very bottom called 'limitations'.